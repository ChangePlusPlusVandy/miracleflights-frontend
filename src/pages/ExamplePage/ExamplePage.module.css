.exampleContainer {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  width: 100%;
}

.exampleTitle {
  font-size: 2rem;
  font-weight: bold;
  color: #000;
  margin-bottom: 1rem;
  font-family: "Roboto", sans-serif;
}
