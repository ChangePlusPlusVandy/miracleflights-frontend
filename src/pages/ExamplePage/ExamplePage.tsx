import styles from "./ExamplePage.module.css";
import ExampleComponent from "../../components/ExampleComponent/ExampleComponent";

const ExamplePage = () => {
  return (
    <div className={styles.exampleContainer}>
      <h1 className={styles.exampleTitle}>Example Page</h1>
      <ExampleComponent incrementAmount={2} />
    </div>
  );
};

export default ExamplePage;
