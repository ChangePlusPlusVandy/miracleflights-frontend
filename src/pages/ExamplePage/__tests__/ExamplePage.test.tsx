import ExamplePage from "../ExamplePage";
import { render } from "@testing-library/react";

/**************************EXAMPLE PAGE TESTING SUITE*******************************/
describe("ExamplePage Tests", () => {
  /**************************SNAPSHOT TEST*******************************/
  test("Renders the ExamplePage snapshot correctly", () => {
    const { container } = render(<ExamplePage />);
    expect(container).toMatchSnapshot();
  });
});
