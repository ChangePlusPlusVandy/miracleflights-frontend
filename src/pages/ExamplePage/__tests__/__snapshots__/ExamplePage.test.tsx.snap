// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExamplePage Tests Renders the ExamplePage snapshot correctly 1`] = `
<div>
  <div
    class="exampleContainer"
  >
    <h1
      class="exampleTitle"
    >
      Example Page
    </h1>
    <div
      class="exampleContainer"
    >
      <div
        class="exampleSubtitle"
      >
        Hit the button to increment the number by 
        2
      </div>
      <div
        class="exampleValueDisplay"
      >
        Current number: 
        0
      </div>
      <div>
        <button
          class="exampleButtonBlue"
        >
          Increment
        </button>
        <button
          class="exampleButtonRed"
        >
          Reset
        </button>
      </div>
    </div>
  </div>
</div>
`;
